src/wazuh_db/helpers/wdb_agents_helpers.c

```diff
@@ -13,13 +13,53 @@
 #include "wazuhdb_op.h"
 
 static const char *agents_db_commands[] = {
+    [WDB_AGENTS_SYS_OSINFO_GET] = "agent %d osinfo get",
     [WDB_AGENTS_VULN_CVES_INSERT] = "agent %d vuln_cves insert %s",
     [WDB_AGENTS_VULN_CVES_UPDATE_STATUS] = "agent %d vuln_cves update_status %s",
     [WDB_AGENTS_VULN_CVES_REMOVE] = "agent %d vuln_cves remove %s",
     [WDB_AGENTS_VULN_CVES_CLEAR] = "agent %d vuln_cves clear"
 };
 
-int wdb_agents_vuln_cves_insert(int id,
+int wdb_agents_sys_osinfo_check_triaged(int id,
```

Comment:
As we are already logging an error message when an error occurred inside of this method: Can we simplify the return value to make it a boolean?

Reply:
I took into account what you are suggesting. But I considered that would be interesting for the caller to have information about the result. That's why the possible values are `1` if triaged, `0` if not triaged or `-1` if error.
In this way, the caller could decide whether to use a default value for the triaged status or abort.

Reply:
Ok. I think it will be useful information.